### DISCUSSION TITLE
Order on the many side

### DISCUSSION THREAD

**Discussion by katrien-c** [PostgREST/postgrest#2469] [Q&A] [answered]
On https://postgrest.org/en/stable/api.html#one-to-many-relationships you'll see an example of a one to many relationships.
I was wondering if postgrest is able to sort on the title of the film ?

I've tried with select=last_name,films(title)&order=films(title).desc so I assume it's not possible ?

**Comment by steve-chavez** [comment, marked as answer]
Check https://postgrest.org/en/stable/api.html#embedded-filters. Should be like

```
select=last_name,films(title)&films.order=title.desc
```
